plenum/server/consensus/view_change_service.py

```diff
@@ -100,7 +101,7 @@ def process_need_view_change(self, msg: NeedViewChange):
         self.view_change_votes.add_view_change(vc, self._data.name)
 
         # 6. Unstash messages for new view
-        self._router.process_all_stashed(STASH_VIEW)
+        self._router.process_all_stashed(STASH_WAITING_NEW_VIEW)
```

Comment:
This one looks suspicious. Comment from @ashcherbakov would be welcome.

Reply:
I think this one needs to be removed, since it's called in `process_new_view_checkpoints_applied` in Ordering service.